packages/tester-python/create-patient.py

```diff
@@ -0,0 +1,22 @@
+from metriport import BaseOrganization, OrgType, Address, UsState
```

Comment:
testing with the fern metriport api still in progress. We can get the code example out to customer though before since not actually something being used by us at all. Just an example / framework